src/D2L.CodeStyle.Analyzers/ApiUsage/DangerousMemberUsages/DangerousMemberUsagesAnalyzer.cs

```diff
@@ -137,6 +146,10 @@ IImmutableSet<ISymbol> dangerousMembers
 			return true;
 		}
 
+		private bool IsDangerousMemberAttribute( ISymbol memberSymbol, INamedTypeSymbol dangerousMemberType ) {
+			return memberSymbol.GetAttributes().Where( attr => attr.AttributeClass.Equals( dangerousMemberType, SymbolEqualityComparer.Default ) ).Any();

```

Comment:
nit: could be slightly simplified:

```suggestion
			return memberSymbol.GetAttributes().Any( attr => attr.AttributeClass.Equals( dangerousMemberType, SymbolEqualityComparer.Default ) );
```